content/docs/security/backend-tls.md

```diff
@@ -166,43 +165,170 @@ Now that your TLS backend and routing resources are configured, verify the TLS c
    HTTP/1.1 200 OK
    ```
 
-3. Enable port-forwarding on the Gateway.
+2. Enable port-forwarding on the Gateway.
 
    ```sh
    kubectl port-forward deploy/http -n kgateway-system 19000
    ```
 
-4. In your browser, open the Envoy stats page at [http://127.0.0.1:19000/stats](http://127.0.0.1:19000/stats).
+3. In your browser, open the Envoy stats page at [http://127.0.0.1:19000/stats](http://127.0.0.1:19000/stats).
 
-5. Search for the following stats that indicate the TLS connection is working. The count increases each time that the Gateway sends a request to the NGINX server.
+4. Search for the following stats that indicate the TLS connection is working. The count increases each time that the Gateway sends a request to the NGINX server.
 
    * `cluster.kube_default_nginx_8443.ssl.versions.TLSv1.2`: The number of TLSv1.2 connections from the Envoy gateway proxy to the NGINX server.
    * `cluster.kube_default_nginx_8443.ssl.handshake`: The number of successful TLS handshakes between the Envoy gateway proxy and the NGINX server.
+   
+## External service
+
+Set up a Backend resource that represents your external service. Then, use a BackendTLSPolicy to instruct the gateway proxy to originate a TLS connection from the gateway proxy to the external service. 
+
+1. Install the experimental channel of the {{< reuse "docs/snippets/k8s-gateway-api-name.md" >}} so that you can use BackendTLSPolicy.
```

Comment:
i think we've been putting this in the prereq conref instead as {{< reuse "docs/snippets/prereq-x-channel.md" >}}